src/main/resources/mesosphere/marathon/api/v2/EventResource_get.md

```diff
@@ -0,0 +1,46 @@
+## GET `/v2/events`
```

Comment:
Actually, I am quite wrong but also quite confused.

Shouldn't this get included in rest-api.md? Do we really duplicate all our rest api documentation across all files?

I just looked at AppsResource_create and saw it doesn't match rest-api.md.

Damn.